CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKHitTester.java

```diff
@@ -95,12 +108,37 @@
     private CVKImage cvkDepthImage = null;
     private Long hFrameBufferHandle = null;
     private CVKCommandBuffer commandBuffer = null;
-    // TODO Use swapchain image format?
-    //private int colorFormat = VK_FORMAT_R8G8B8A8_UINT;
-
-    // ========================> Static init <======================== \\
-    
-    
+    private final int colorFormat = VK_FORMAT_R32_SFLOAT;   // Only use the red channel for hit testing
```

Comment:
How does the JOGL version support more than 255 icons?  Looking at the shader, the hit test value is the vec4 hbv made in the geometry shader:
hbv = vec4(gData[0][3] + 1, 0, 0, 1);
That implies only a single byte used for ids.   What am I missing?

Reply:
no its an int = 4 bytes

Reply:
tested and works for sphere builder with 10000 icons